examples/experimental/onnx/yolov5/0001-Add-NNCF-ONNX-PTQ-example-notebook.patch

```diff
@@ -969,11 +969,11 @@ index 0000000..96bd204
 +   "source": [
 +    "import onnx\n",
 +    "\n",
-+    "from nncf.experimental.post_training.compression_builder import CompressionBuilder\n",
-+    "from nncf.experimental.post_training.algorithms.quantization import PostTrainingQuantization\n",
-+    "from nncf.experimental.post_training.algorithms.quantization import PostTrainingQuantizationParameters\n",
++    "from nncf.quantization.compression_builder import CompressionBuilder\n",
++    "from nncf.quantization.algorithms import PostTrainingQuantization\n",
```

Comment:
Why is it different methods in different samples: `DefaultQuantization` and `PostTrainingQuantization`?

Reply:
Fixed, thanks.